gcp-deployment-manager/project.env

```diff
@@ -0,0 +1,19 @@
+# Project constants. DO NOT OVERRIDE.
+CLOUD_SDK_IMAGE_VERSION=298.0.0-alpine
+DISK_DEPLOYMENT=docksal-sandbox-disk
+GROUP_DEPLOYMENT=docksal-sandbox-group
+GROUP_NAME=mig-docksal-group-py
+TEMPLATE_DEPLOYMENT=docksal-sandbox-template
+
+# Overridable properties.
+# NOTE: If changing PREEMPTIVE, ZONE, or DEFAULT_MACHINE_SIZE then you MUST
+# also change TEMPLATE_ID to be something different as well. We recommend
+# chaning it to e.g. custom-docksal-sandbox-template-002. Always increment the
+# last number as a matter of convention.
+ZONE=us-east4-c
+REGION=us-east4
+DEFAULT_MACHINE_SIZE=n1-standard-2
+DISK_SIZE=150
+TEMPLATE_ID=docksal-sandbox-template-001
+PREEMPTIVE=true
+DOCKSAL_VERSION=develop
```

Comment:
@lmakarov Revert to master and also bump image to ubuntu 20.04?

Reply:
Yes, please